Source/CarthageKit/Dependency.swift

```diff
@@ -23,7 +41,7 @@ public enum Dependency {
 			return url.name ?? url.urlString
 
 		case let .binary(url):
-			return url.lastPathComponent.stripping(suffix: ".json")
+			return url.url.lastPathComponent.stripping(suffix: ".json")
```

Comment:
Can me move this logic into a method on `BinaryURL`?

Reply:
Will address in upcoming commit